### PR TITLE
Editorial: Add jib as editor

### PR DESCRIPTION
**Pull request by adam-be**

### CODE REVIEW

File: webrtc.js [hunk 1]
```diff
@@ -65,6 +65,9 @@ var respecConfig = {
     },
     { name: "Taylor Brandstetter", company: "Google",
       w3cid: "82908"
+    },
+    { name: "Jan-Ivar Bruaroey", company: "Mozilla",
```
Comment (alvestrand) [inline webrtc.js]: Can we use ø here, or does he prefer to be Bruaroey?

Comment (adam-be) [reply webrtc.js]: I took the same from the W3C records.

Comment (adam-be) [reply webrtc.js]: @jan-ivar, any preference on this topic? :)

Comment (jan-ivar) [reply webrtc.js]: I use Jan-Ivar Bruaroey.

**Pull request merged**